frappe/database/postgres/database.py

```diff
@@ -416,6 +416,33 @@ def get_column_type(self, doctype, column):
 	def get_database_list(self):
 		return self.sql("SELECT datname FROM pg_database", pluck=True)
 
+	def get_column_index(
+		self, table_name: str, fieldname: str, unique: bool = False, order: int = 1
```

Comment:
MariaDB variant doesn't take in `order` and no value is being passed when this is being called, so maybe would make sense to hardcode instead of the additional parameter?